### PR TITLE
Add a field to store estimated hours for bounties

### PR DESCRIPTION
**Pull request by kn**
##### Description

This PR adds `estimated_hours` field to `dashboard.Bounty` model for future use to predict bounty success rates as described in issue #2117.

The field is nullable for backward compatibility so that we don't need to specify an arbitrary value for existing bounties. We could make 0 as a default value instead if preferred but null seems cleaner to indicate "not provided".

##### Checklist

- [x] linter status: 100% pass
- [x] changes don't break existing behavior
- [x] commit message follows [commit guidelines](https://docs.gitcoin.co/mk_contributors/#step-4-commit)

##### Testing

* Added a test case and it passes.
* Manually tested via Django admin by creating a bounty and updating existing bounties.

##### Refers/Fixes

Refs: https://github.com/gitcoinco/web/issues/2117

### CODE REVIEW

File: app/dashboard/tests/test_dashboard_models.py [hunk 1]
```diff
@@ -76,7 +77,7 @@ def test_bounty():
             bounty=bounty,
             profile=fulfiller_profile,
         )
-        assert str(bounty) == f'5: foo, 0 ETH @ {naturaltime(bounty.web3_created)}'
+        assert str(bounty) == f'{bounty.pk}: foo, 0 ETH @ {naturaltime(bounty.web3_created)}'
```
Comment (kn) [inline app/dashboard/tests/test_dashboard_models.py]: FYI: made this test more robust by avoiding hard coding implicit pk value.

File: app/dashboard/models.py [hunk 2]
```diff
@@ -221,6 +221,7 @@ class Bounty(SuperModel):
     token_address = models.CharField(max_length=50)
     bounty_type = models.CharField(max_length=50, choices=BOUNTY_TYPES, blank=True)
     project_length = models.CharField(max_length=50, choices=PROJECT_LENGTHS, blank=True)
+    estimated_hours = models.IntegerField(blank=True, null=True)
```
Comment (mbeacom) [inline app/dashboard/models.py]: Should we use a `PositiveIntegerField` here?

Comment (kn) [reply app/dashboard/models.py]: great idea! updated

**Comment by kn** [comment]
@mbeacom You are right. I misunderstood a description of the issue. I've added an additional commit that does the following:
* The frontend stores estimated hours as a metadata in IPFS.
* The backend picks up the estimated hours stored in the metadata and store the value in Postgres DB when sync with web3.

Please take another look. Thanks!

**Comment by kn** [comment]
Thanks for the reviews!

CircleCI is failing and I think it's due to cache. Would anyone be able to retrigger CI without cache if you have permission to do so?

**Comment by kn** [comment]
Also it looks like I can't merge unless I have at least one approved review :)

**Comment by frankchen07** [comment]
@kn, hopefully @mbeacom will get around to this soon, he's currently in the midst of working on high-priority launch items.

**Comment by frankchen07** [comment]
hi @kn - sorry for the delay, looking into getting some reviewers for this!

**Comment by kn** [comment]
Thanks! I'll take you up on that ;)

**Pull request merged**